api/api.py

```diff
@@ -27,11 +27,11 @@ def home():
 #TODO: Implement way to query DB and put all employees in an array like the employees array above to display in API response
 
 # A route to return all employees.
-@app.route('/api/v1/employees/all', methods=['GET'])
+@app.route('/api/v1/employees', methods=['GET'])
 def api_all():
     return jsonify(employees)
 
-@app.route('/api/v1/employees', methods=['GET'])
+@app.route('/api/v1/employees/id', methods=['GET'])
```

Comment:
I should have probably worded this differently in Discord. The endpoint to find a specific employee should be `/api/v1/employees/{employee_id)`. Where employee_id is the employee_id attribute of the employee you are looking for.

Reply:
Fixed it up. Let me know what you think!